### ISSUE TITLE
NagBar Picks Wrong Font Color Sometimes on Sonoma

### ISSUE THREAD

**Issue by timkite** [volendavidov/NagBar#18]
As of macOS Sonoma, on certain wallpaper/dark mode vs. light mode combinations, NagBar will pick the wrong font color to display. This may have happened in Ventura too, but hey, there are new wallpapers.

For example, the "Grand Canyon River Valley" wallpaper will generate a light colored menu bar even in dark mode. NagBar gets confused and uses white text when black text is correct.
<img width="278" alt="Screenshot 2023-10-04 at 20 39 37" src="https://github.com/volendavidov/NagBar/assets/6933976/6f6791f7-5c7b-494e-a1c2-8887b06124fd">

**Comment by volendavidov** [comment]
The text is white regardless whether I use light or dark mode (as well as the rest of the menu items; their color remains white). Perhaps there is another setting that makes the menubar items dark?

<img width="163" alt="Screenshot 2024-09-02 at 20 23 27" src="https://github.com/user-attachments/assets/52819a2c-5b0c-40b0-af13-c2ee798af8d2">

**Comment by timkite** [comment]
Right, that's what I'm seeing from NagBar, whereas other menu bar items will adjust their color between black and white depending on the primary background color behind the menu bar (I'm not sure if light or dark mode factors in unless that changes the background color too). I've worked around the issue by picking a wallpaper that's always "dark enough" to trigger white text in the menu bar regardless of whether the system is in light or dark mode.
